src/kind.ts

```diff
@@ -0,0 +1,74 @@
+import { Job } from "@brigadecore/brigadier";
+
+export const kindJobImage = "radumatei/golang-kind:1.11-0.4";
+
+export class KindJob extends Job {
+    constructor(name: string, image?: string) {
+        if (image == undefined) {
+            image = kindJobImage;
+        }
+        super(name, image);
+
+        // kind needs to run as a privileged pod
+        this.privileged = true;
```

Comment:
Since setting up a `kind` cluster takes a long time, we should probably set the timeout to be larger by default.